src/stream/src/executor/debug/cache_clear.rs

```diff
@@ -18,19 +18,19 @@ use std::sync::Once;
 use async_trait::async_trait;
 use risingwave_common::error::Result;
 
-use crate::executor::{Executor, Message};
+use crate::executor::{ExecutorV1, Message};
 
 pub const CACHE_CLEAR_ENABLED_ENV_VAR_KEY: &str = "RW_NO_CACHE";
 
 /// [`CacheClearExecutor`] clears the memory cache of `input` executor after a barrier passes.
 #[derive(Debug)]
 pub struct CacheClearExecutor {
```

Comment:
Will this Executor work now? If not, we can remove it first.

Reply:
NTFS!